notebooks/niriss_ami_binary/3_niriss_ami_binary.ipynb

```diff
@@ -0,0 +1,627 @@
+{
```

Comment:
<p>Could you add <code>from pathlib import Path</code> to the import cell and replace the matching lines from this cell with the following code?</p><p> </p><pre class="ql-syntax" spellcheck="false">boxfile = Path('./niriss_ami_binary3.zip')

# Download zip file
if not os.path.exists(boxfile):
</pre><p> </p><p>Defining the path this way should be safer for users on non-Unix machines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>